source/installguide/management-server/_database.rst

```diff
@@ -331,6 +351,14 @@ same node for MySQL. See `“Install the Database on the Management Server Node
 
          service iptables restart
 
+   .. warning::
+      On CentOS 8 / SUSE, firewalld is the default firewall manager and controls iptables. It is
+      recommended that it be disabled ``systemctl stop firewalld ; systemctl disable firewalld``
```

Comment:
Why? Just explanation - I can agree, since in production (in past) I had all local firewall turned off, in small cloud env, since we aggressively protected the access from the outside, so internal VM-level firewall was completely unneeded and hassle to manage)

Reply:
Added the reason as well